libexec/probes/worker-scripts/uploader/activemquploader.py

```diff
@@ -38,11 +38,12 @@ def publishSToMq(self, arguments, event_types, summaries, summaries_data):
             msg_body['summaries'] = summaries_data[event]
             msg = Message(body=json.dumps(msg_body), header=msg_head)
             size_msg = msg.size()
-            # if size of the message is larger than 10MB discarrd
-            if size_msg > size_limit:
+            self.add2log("Message size: %s" % size_msg)

```

Comment:
Do you really want that log line for every message sent?

Reply:
opps this was a prolbme I fixed it. Good catch.